test/modules/TestResultCoordinator/docker/linux/arm64v8/Dockerfile

```diff
@@ -12,7 +12,5 @@ COPY $EXE_DIR/ ./
 # Expose HTTP port
 EXPOSE 5001/tcp
 
-USER moduleuser
```

Comment:
why remove?

Reply:
This is removed because the module-full base image has elevated permissions.